src/main/resources/dev/definitions/prisons/orphanage/dpd005-court-movements.json

```diff
@@ -112,24 +112,28 @@
           {
             "name": "$ref:date",
             "display": "Date",
-            "formula": "",
-            "visible": "mandatory",
+            "formula": "format_date(${date}, 'dd/MM/yyyy')",
+            "visible": "true",
```

Comment:
Just double checking the intention here. 
Is the date filter intended not to be mandatory?